src/content/docs/tutorial-innovation-and-growth/development-quality-mgmt.mdx

```diff
@@ -0,0 +1,166 @@
+
+You can find the KPIs and minimum-required attributes needed for this use case in the [key performance indicators](#key-perf-indicators) section above. Typically, you'll use your development toolchain's APIs to extract the KPIs and their attributes, then submit them to New Relic using the [custom events API](/docs/data-apis/ingest-apis/introduction-event-api/).
+
+Before starting any custom integration work, you should find if any applicable out-of-box integrations exist.    
+    </Step>
+    <Step>
+### Implement dashboards [#implement-dashboards]
+
+Our <InlinePopover type="dashboards" /> are the primary method this improvement process uses. They show your current KPIs and help you to identify the areas which require improvement. You can find sample dashboards [in the New Relic OMA resource center on GitHub](https://github.com/newrelic/oma-resource-center).
+
+The information displayed in the dashboards depends on your development toolchain, so you'll need to [customize your dashboard](https://developer.newrelic.com/collect-data/monitor-your-application/query-custom-data/) to your exact specifications.      
+    </Step>
+    <Step>
+### Establish a baseline [#establish-baseline]
+
+Because you need enough data to form a baseline before you can [perform the initial enablement](#perform-enablement-one), you must establish your baselines that consist of a sample of development activity. Normally, this will be a minimum of two weeks, but it can be up to six weeks depending on your current development pace. One easy way to do this is to align your baseline collection and evaluation cycle with your Agile sprints, if applicable.
+
+You should periodically ensure that event data is accumulating as expected in New Relic while you establish your baselines.        
+    </Step>
+    <Step>
+### Meet with your team [#meet-with-team]
+
+After establishing your baselines, you'll introduce development teams and other stakeholders to the collected data and the ongoing continuous improvement process you'll be following.
+
+The process consists of three activities:
+1. **Review the KPIs and trends**: You and the stakeholders will look at the KPIs and identify trends.
+2. **Identfy achievements, challenges, and opportunities**: In this phase, you'll identify areas where KPIs are improving (achievements) and areas where they're not improving (challenges). You'll then identify strategies and tactics for improving KPIs (opportunities) and how best to implement them.
+3. **Make technical recommendations**: Here, you and the relevant stakeholders will identify and review technical recommendations, such as making changes to your development toolchain or observability strategies.        
+    </Step>
+    <Step>
+### Improvement process [#improvement-process]
```

Comment:
To follow the style of the previous steps, maybe:

`Review your progress against baselines`

Reply:
Good catch, I'll make these more active. Thanks!